pkg/component/web.go

```diff
@@ -108,8 +108,7 @@ func (c *Component) initWeb() error {
 				webmiddleware.AuthUser(healthUsername, c.config.HTTP.Health.Password),
 			)))
 		}
-		g.HandleFunc("/healthz/live", c.healthHandler.LiveEndpoint)
-		g.HandleFunc("/healthz/ready", c.healthHandler.ReadyEndpoint)
+		g.Handle("/healthz", c.healthHandler.GetHandler())
```

Comment:
Check if the Docker health status checks still works. It still uses the old URL:

https://github.com/TheThingsNetwork/lorawan-stack/blob/8a7bcd6c98c6757a2dd36f8090a4fdc8478be704/Dockerfile#L40

Reply:
Tested locally - works